docs/components/modeler/web-modeler/modeling-dmn.md

```diff
@@ -0,0 +1,49 @@
+---
+id: modeling-dmn
+title: Modeling DMN
+description: Let's create a DMN diagram.
+---
+
+## Create new DMN decision requirement diagram
+
+![New DMN Diagram](./img/dmn/create-dmn.png)
+
+To start modeling, create a new DMN diagram by selecting **New > DMN Diagram** in your project.
+
+## Start modeling
+
+![Start Modeling](./img/dmn/main.png)
```

Comment:
The rest should be covered as @akeller mentioned by @MaxTru's PR: https://github.com/camunda-cloud/camunda-cloud-documentation/pull/738#pullrequestreview-932578226